lib/read-theme.js

```diff
@@ -66,10 +66,17 @@ readThemeStructure = function readThemeFiles(themePath, subPath, arr) {
 };
 
 readHbsFiles = function readHbsFiles(theme) {
+    const themeFilesContent = _.filter(theme.files, function (themeFile) {
+        if (themeFile.ext) {
+            return themeFile.ext.match(/\.hbs|\.css/) || themeFile.file.match(/package.json/);
+        }
+    });
     // Setup a partials array
     theme.partials = [];
 
-    return Promise.map(_.filter(theme.files, {ext: '.hbs'}), function (themeFile) {
+    // CASE: we need the actual content of all css, hbs files, and package.json for
+    // our checks
+    return Promise.map(themeFilesContent, function (themeFile) {
```

Comment:
> themeFile.compiled = readHbs.handlebars.precompile(content);

We don't even have to try to precompile if it's not a css file.
Maybe use two functions? `readHbsFiles` and `readCssFiles`?